DeepSpeech.py

```diff
@@ -1438,85 +1438,89 @@ def end(self, session):
     # The MonitoredTrainingSession takes care of session initialization,
     # restoring from a checkpoint, saving to a checkpoint, and closing when done
     # or an error occurs.
-    with tf.train.MonitoredTrainingSession(master='' if server is None else server.target,
-                                           is_chief=is_chief,
-                                           hooks=hooks,
-                                           checkpoint_dir=FLAGS.checkpoint_dir,
-                                           save_checkpoint_secs=FLAGS.checkpoint_secs,
-                                           config=session_config) as session:
+    try:
+        with tf.train.MonitoredTrainingSession(master='' if server is None else server.target,
+                                               is_chief=is_chief,
+                                               hooks=hooks,
+                                               checkpoint_dir=FLAGS.checkpoint_dir,
```

Comment:
According to `pydoc tensorflow.train.MonitoredTrainingSession`, the `checkpoint_dir` is optionnal and `save_checkpoint_secs` can be set to `None` to disable the default checkpoint saver.